sys/cheri/cheri.h

```diff
@@ -97,9 +97,12 @@ void * __capability	_cheri_capability_build_user_rwx(uint32_t perms,
  * Global capabilities used to construct other capabilities.
  */
 
-/* Root of all userspace capabilities. */
+/* Root of all unsealed userspace capabilities. */
```

Comment:
Is it accurate to say this is the root? I feel like it's the sealing authority and userspace_cap is the root.

Reply:
That makes sense.